cl_manycore/libraries/bsg_manycore_cuda.cpp

```diff
@@ -1250,13 +1591,18 @@ int hb_mc_device_free (hb_mc_device_t *device, eva_t eva) {
 /**
  * Copies a buffer from src on the host/device DRAM to dst on device DRAM/host.
  * @param[in]  device        Pointer to device
- * @parma[in]  src           EVA address of src 
+ * @parma[in]  src           EVA address of source to be copied from
+ * @parma[in]  dst           EVA address of destination to be copied into
  * @param[in]  name          EVA address of dst
- * @param[in]  count         Size of buffer to be copied
+ * @param[in]  count         Size of buffer (number of words) to be copied
  * @param[in]  hb_mc_memcpy_kind         Direction of copy (HB_MC_MEMCPY_TO_DEVICE / HB_MC_MEMCPY_TO_HOST)
  * @return HB_MC_SUCCESS if succesful. Otherwise an error code is returned.
  */
-int hb_mc_device_memcpy (hb_mc_device_t *device, void *dst, const void *src, uint32_t count, enum hb_mc_memcpy_kind kind) {
+int hb_mc_device_memcpy (	hb_mc_device_t *device,
```

Comment:
@bornaehsani can you remove tabs from the source files you edited, and tabs like here -- where they're between the parenthesis and the first argument? This is inconsistent with the rest of our source code